src/cov.jl

```diff
@@ -58,66 +58,41 @@ cov
 
 
 """
-    mean_and_cov(x, [wv::WeightVec]; vardim=1) -> (mean, cov)
+    mean_and_cov(x, [wv::AbstractWeights]; vardim=1) -> (mean, cov)
```

Comment:
Yeah, I was holding off updating these docstrings until I was more comfortable with the behaviour. Since, folks don't seem to have an issue with the `corrected=false` keyword for all these functions I'll update that.